content/2-templating/3-loop/emberPolaris/colors.gjs

```diff
@@ -0,0 +1,9 @@
+import { array } from "@ember/helper";
+
+<template>
+  <ul>
+    {{#each (array "red" "green" "blue") as |color|}}
```

Comment:
why not define the array in JS, above the template?

Reply:
I think it's much better than importing `array`! To be honest I wasn't even aware that'd work without going for an entire `Component` class. :smile: